### PR TITLE
Add a note about collaboration on atlas standards

### PR DESCRIPTION
**Pull request by adamltyson**
@PolarBean feel free to suggest any changes. Happy to add any logos to the funders page too.

### CODE REVIEW

**Comment by PolarBean** [comment]
and link to https://www.ebrains.eu/national-nodes/norway

File: docs/source/about.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 # About
 
 ## Introduction
-The BrainGlobe Initiative (BGI) was established by researchers at the
+The BrainGlobe Initiative (BGI) was originally established by researchers at the
 [Sainsbury Wellcome Centre](https://www.sainsburywellcome.org/web/) and the 
 [Technical University of Munich](https://www.tum.de/en/). 
 
+We collaborate with the [EBRAINS](https://www.ebrains.eu/) national node at the [University of Oslo](https://www.uio.no/english/) to establish and implement standards for atlases and their metadata.
```
Comment (adamltyson) [inline docs/source/about.md:8]:
```suggestion
We collaborate with researchers from the [EBRAINS national node](https://www.ebrains.eu/national-nodes/norway) at the [University of Oslo](https://www.uio.no/english/) to establish and implement standards for atlases and their metadata.
```

**Comment by adamltyson** [comment]
How's this version?

**Comment by PolarBean** [comment]
Good! I ran it by Jan and he asked if we could have it as the following 
"We collaborate with the [EBRAINS Norwegian node](https://www.ebrains.eu/national-nodes/norway) based at [the University of Oslo](https://www.uio.no/english/), to establish and implement standards and tools for atlases and their metadata."

Also it would be cool to have the UiO logo on the funders page, Jan has to double check before adding the EBRAINS logo. Let me know if this all sounds OK or if it's too much.

**Review by PolarBean — CHANGES_REQUESTED** [review]
We collaborate with researchers from the [EBRAINS Norwegian node](https://www.ebrains.eu/national-nodes/norway) based at [the University of Oslo](https://www.uio.no/english/), to establish and implement standards and tools for atlases and their metadata

**Comment by PolarBean** [comment]
both dark and light mode logos are here: https://www.uio.no/english/about/designmanual/profile-elements/logo/

**Comment by adamltyson** [comment]
@PolarBean I think this should be good. Happy to add the EBRAINS logo at a later date.

**Comment by PolarBean** [comment]
Great! I'm happy for it to be merged

**Pull request merged**